integ-test-promote.pipeline.yml

```diff
@@ -207,6 +207,46 @@ stages:
               grep -v FirstTest test-results/matlab/selectbytag.xml
               grep -v simpleTest test-results/matlab/selectbytag.xml
             displayName: Verify test filtered by tag name
+          - bash: |
+              echo 'diary console.log' >> startup.m
+            displayName: Setup diary for logging
+          - task: MathWorks.matlab-azure-devops-extension-dev.RunMATLABTests.RunMATLABTests@0
+            displayName: Run MATLAB tests with strict checks
+            inputs:
+              strict: true
+              sourceFolder: src
+          - bash: |
+              set -e
+              grep -q "runner.addPlugin(FailOnWarningsPlugin())" console.log
+            displayName: Verify tests ran with strict checks
+          - task: MathWorks.matlab-azure-devops-extension-dev.RunMATLABTests.RunMATLABTests@0
+            displayName: Run MATLAB tests in parallel
+            inputs:
+              useParallel: true
+              sourceFolder: src
+          - bash: |
+              set -e
+              grep -q "Starting parallel pool (parpool)" console.log
+            displayName: Verify tests ran in parallel
+            condition: startsWith(variables['Agent.Name'], 'vmss')
+          - task: MathWorks.matlab-azure-devops-extension-dev.RunMATLABTests.RunMATLABTests@0
+            displayName: Run MATLAB tests with maximum display level for event details
```

Comment:
Is the word MATLAB necessary?

Reply:
The task names were chosen to follow the same pattern as existing test tasks. For eg: tasks starting from line 172 in [integ-test-promote.pipeline.yml](https://github.com/mathworks/matlab-azure-devops-extension/blob/master/integ-test-promote.pipeline.yml)